### PR TITLE
add emax elrs config

### PR DESCRIPTION
**Pull request by EmaxModel**

### CODE REVIEW

File: configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+set current_meter = ADC
+set battery_meter = ADC
+set ibata_scale = 179
+set beeper_inversion = ON
+set beeper_od = OFF
+set system_hse_mhz = 8
+set max7456_spi_bus = 2
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180
```
Comment (haslinghuis) [inline configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config:111]:
```suggestion
set gyro_1_sensor_align = CW180

```

File: configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+set expresslrs_domain = ISM2400
```
Comment (haslinghuis) [inline configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config:112]: Good call to include ISM2400 domain.

Add new line please

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config (hunk 1 above)
Comment (EmaxModel) [reply configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config]: Is there anything different?

**Review by haslinghuis — DISMISSED** [review]

File: configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config [hunk 3]
```diff
@@ -0,0 +1,112 @@
+# Betaflight / STM32F411SX1280 (S4SX) 4.3.1 Jul 13 2022 / 03:33:32 (8d4f005) MSP API: 1.44
+
+#define USE_GYRO_SPI_MPU6000
+#define USE_ACC_SPI_MPU6000
+#define USE_ACCGYRO_BMI270
+#define USE_GYRO_SPI_ICM20689
+#define USE_ACC_SPI_ICM20689
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+#define USE_RX_SPI
```
Comment (haslinghuis) [inline configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config:10]: Can be removed as we already enable feature RX_SPI below

File: configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+#define USE_RX_EXPRESSLRS
+#define USE_RX_EXPRESSLRS_TELEMETRY
+#define USE_RX_SX1280
+#define USE_MAX7456
+
+board_name EMAX_TINYHAWK_F411_ELRS
```
Comment (haslinghuis) [inline configs/default/EMAX-EMAX_TINYHAWK_F411_ELRS.config:16]: Should we just call it EMAX_TINYHAWKF4SX1280 like other ELRS SPI targets?

**Review by haslinghuis — APPROVED** [review]

File: configs/default/EMAX-EMAX_TINYHAWKF4SX1280.config [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+dma pin A00 0
+# pin A00: DMA1 Stream 2 Channel 6
+dma pin A09 0
+# pin A09: DMA2 Stream 6 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+
+# feature
+feature RX_SPI
+feature OSD
```
Comment (haslinghuis) [inline configs/default/EMAX-EMAX_TINYHAWKF4SX1280.config:95]: Some targets add `feature TELEMETRY` - and some don't - so not sure if that is needed.

**Review by haslinghuis — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]
LGTM !

**Review by blckmn — APPROVED** [review]

**Pull request merged**